tools/cli/pkg/ers/command/status.go

```diff
@@ -0,0 +1,258 @@
+	ersClient, err := client.NewErsClient(ers.GlobalOpts.ErsUrl())
+	if err != nil {
+		return errors.Wrap(err, "while initializing ers client")
+	}
+	defer ersClient.Close()
+
+	instance, err := ersClient.GetOne(c.instanceID)
+	if err != nil {
+		return errors.Wrap(err, "while getting ers instance")
+	}
+
+	jsonData, err := ToJson(instance)
+	if err != nil {
+		return errors.Wrapf(err, "while creating a json")
+	}
+	fmt.Printf("%s - %s\n", instance.State, instance.StateMessage)
+	fmt.Printf("Migrated - %t\n", instance.Migrated)
+	c.log.Debugf(jsonData)
+
+	// KEB
+	fmt.Printf("%sKEB%s - ", Red, Reset)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	auth := command.CLICredentialManager(c.log)
+	httpClient := oauth2.NewClient(ctx, auth)
+
+	kebClient := runtime.NewClient(ers.GlobalOpts.KebApiUrl(), httpClient)
+
+	runtimesPage, err := kebClient.ListRuntimes(runtime.ListParameters{InstanceIDs: []string{c.instanceID}})
+	if err != nil {
+		return errors.Wrap(err, "while listing runtimes")
+	}
+
+	if len(runtimesPage.Data) != 1 {
+		fmt.Printf("No data in KEB")
+	}
+
+	runtime := runtimesPage.Data[0]
+	jsonData, err = ToJson(runtime)
+	if err != nil {
+		return errors.Wrapf(err, "while creating a json")
+	}
+	fmt.Printf("%s\n", runtime.Status.State)
+	fmt.Printf("Provisioning: %s %s\n", runtime.Status.Provisioning.State, runtime.Status.Provisioning.CreatedAt)
+
+	if runtime.Status.UpgradingCluster != nil {
+		for _, v := range runtime.Status.UpgradingCluster.Data {
+			fmt.Printf("Upgrade Cluster: %s %s\n", v.State, v.CreatedAt)
+		}
+	}
+
+	if runtime.Status.UpgradingKyma != nil {
+		for _, v := range runtime.Status.UpgradingKyma.Data {
+			fmt.Printf("Upgrade Kyma: %s %s\n", v.State, v.CreatedAt)
+		}
+	}
+
+	if runtime.Status.Update != nil {
+		for _, v := range runtime.Status.Update.Data {
+			fmt.Printf("Update Kyma: %s %s\n", v.State, v.CreatedAt)
+		}
+	}
+
+	if runtime.Status.Suspension != nil {
+		for _, v := range runtime.Status.Suspension.Data {
+			fmt.Printf("Suspensions: %s %s\n", v.State, v.CreatedAt)
+		}
+	}
+
+	if runtime.Status.Unsuspension != nil {
+		for _, v := range runtime.Status.Unsuspension.Data {
+			fmt.Printf("Unsuspension: %s %s\n", v.State, v.CreatedAt)
+		}
+	}
+
+	if runtime.Status.Deprovisioning != nil {
+		fmt.Printf("Deprovisiong: %s %s\n", runtime.Status.Deprovisioning.State, runtime.Status.Deprovisioning.CreatedAt)
+	}
+	c.log.Debugf("%s", jsonData)
+
+	// Mothership
+	fmt.Printf("%sMOTHERSHIP%s", Red, Reset)
+	mothershipClient, err := mothershipAuth.NewClient(ers.GlobalOpts.MothershipApiUrl(), httpClient)
+	if err != nil {
+		return errors.Wrap(err, "while creating mothership client")
+	}
+
+	response, err := mothershipClient.GetClustersState(ctx, &mothership.GetClustersStateParams{
+		RuntimeID: &runtime.RuntimeID,
+	})
+	if err != nil {
+		return errors.Wrap(err, "wile getting cluster state")
+	}
+
+	defer response.Body.Close()
+
+	if IsErrResponse(response.StatusCode) {
```

Comment:
nit: do we want to handle 204, 205, or 206 given this tests allows `200 <= status < 300`? I would consider making this just `response.StatusCode != http.StatusOK` if it's sufficient. Just wondering if for example ERS can send partial content with 206 and expect we fetch the rest in follow-up calls, but as currently implemented we proceed as if the content was complete.

Reply:
This code snippet does not refer to ERS call.